src/main/java/io/vertx/howtos/ebservice/BeersApplication.java

```diff
@@ -0,0 +1,17 @@
+package io.vertx.howtos.ebservice;
+
+import io.vertx.core.Vertx;
+
+public class BeersApplication {
+
+  public static void main(String[] args) {
+    Vertx vertx = Vertx.vertx();
+    vertx.deployVerticle(new BarmanVerticle(), ar -> {
```

Comment:
You may simplify with a deploy without callback here.

Reply:
DrunkVerticle needs the BarmanVerticle deployed (with BarmanService up and running)